store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9651,4 +9651,13 @@ TODO: delete them permanently from here
         This attribute does not affect data source SMTP connections as those are controlled by zimbraDataSourceSmtpHost, zimbraDataSourceSmtpConnectionType, etc.
   </desc>
 </attr>
+
+<attr id="5005" name="zimbraCAPTCHAEnabled" type="boolean" cardinality="single" optionalIn="account" flags="domainAdminModifiable" since="8.8.6">
```

Comment:
the flag should have "accountInherited" and optionalIn should be "account,cos".
See also a similar attr, zimbraPasswordLockoutEnabled.